### DISCUSSION TITLE
Oracle Database Support

### DISCUSSION THREAD

**Discussion by ionsource** [oqtane/oqtane.framework#1503] [General]
Oracle is not currently a supported database option. Is there any information / documentation related to the multi-database support work done for the 2.1 release that would help me in extending Oqtane to support Oracle?

**Comment by sbwalker** [comment]
There is no documentation available. You would need to create an Oracle database provider by following the other database provider project examples in this repo ( SqlServer, MySQL, SQLite, PostgreSQL ). You would also need to create a razor component for capturing the database parameters in the UI ( see Oqtane.Client\Installer\Controls\ ). Currently the list of supported databases is static - defined in appsettings.json - so you would need to add your Oracle provider to the list ).
